frontend/src/components/mine/MineHeader.js

```diff
@@ -117,6 +126,11 @@ export class MineHeader extends Component {
     this.setState({ menuVisible: false });
   };
 
+  handleClearStatusDate = () => {
+    const date = new Date();
+    this.props.change(FORM.MINE_RECORD, "status_date", date);
+  };
+
```

Comment:
This can occur inside the form component, inside the if/else statement

Reply:
I have questions about implementation for this.  Can we chat when you get in?